### PR TITLE
feature: blob

### PR DESCRIPTION
**Pull request by alimoncul**

### CODE REVIEW

File: global/blob/blob.ts [hunk 1]
```diff
@@ -9,8 +11,8 @@
  *     var myBase64Str = "SGVsbG8gV29ybGQ=";
  *     var blob = Blob.createFromBase64(myBase64Str);
  */
-declare class Blob {
-  constructor(parts: string[], properties: { type: string });
+export declare class Blob extends NativeComponent {
```
Comment (cenkce) [inline global/blob/blob.ts:14]: use interface instead of

File: global/blob/blob.ios.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import NativeComponent from 'core/native-component';
+
+class BlobIOS extends NativeComponent {
```
Comment (cenkce) [inline global/blob/blob.ios.ts:3]: implement interface IBlob

Comment (cenkce) [reply global/blob/blob.ios.ts:3]: lets discuss this

Comment (alimoncul) [reply global/blob/blob.ios.ts]: @cenkce can you check again

**Pull request merged**